launch/tier4_localization_launch/config/ndt_scan_matcher.param.yaml

```diff
@@ -62,3 +62,9 @@
         0.0,   0.0,   0.0,   0.0,      0.000625, 0.0,
         0.0,   0.0,   0.0,   0.0,      0.0,      0.000625,
       ]
+
+    # GNSS Regularization switch
```

Comment:
@KYabuuchi 
The poses provided for regularization are not limited to GNSS. 
I don't think you need to add the word "GNSS".

Reply:
Thanks for your reviewing!! I removed the word in [5ef69cf](https://github.com/autowarefoundation/autoware.universe/pull/1006/commits/5ef69cffe40ee8df703a1749542fdf5526b50684)